src/CSS/CSSRules.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons+Round"
+    rel="stylesheet">
+    <!-- <link rel="stylesheet" href="https://tangent-ui.netlify.app/components/component.css" referrerpolicy="no-referrer"> -->
```

Comment:
Files have many unnecessary comments. If these are not useful then remove these. Classnames are descriptive and simple. Great work.

Reply:
okay removed